src/network/nokia/isam/snmp/mode/hardware.pm

```diff
@@ -88,20 +88,19 @@ You can also exclude items from specific instances: --filter=cardtemperature,108
 
 Define the expected status if no components are found (default: critical).
 
-
 =item B<--threshold-overload>
 
 Use this option to override the status returned by the plugin when the status label matches a regular expression (syntax: section,[instance,]status,regexp).
 Example: --threshold-overload='sfp,OK,^los$'
 
 =item B<--warning>
 
-Set warning threshold for 'sfp.temperature', 'cardtemperature' (syntax: type,regexp,threshold)
+Define the warning thresholds for 'sfp.temperature', 'cardtemperature' (syntax: type,regexp,threshold).
 Example: --warning='sfp.temperature,1.1,30'
 
 =item B<--critical>
 
-Set critical threshold for 'sfp.temperature', 'cardtemperature' (syntax: type,regexp,threshold)
+Define the critical thresholds for 'sfp.temperature', 'cardtemperature' (syntax: type,regexp,threshold).
 Example: --warning='sfp.temperature,1.1,40'
```

Comment:
I don't anderstand this exemple, the regex is 1.1 ? there is no catching group in the regexp ?